cypress/e2e/checkout.cy.js

```diff
@@ -1,11 +1,36 @@
 /// <reference types='cypress' />
 
-describe('', () => {
-  before(() => {
+import HomeAndCataloguePageObject from
+  '../support/pages/homeСatalogue.pageObject';
+import CartPageObject from '../support/pages/cart.pageObject';
+import ContactFormPageObject from '../support/pages/contactForm.pageObject';
 
+const homeAndСatalogue = new HomeAndCataloguePageObject();
+const cartPage = new CartPageObject();
+const contactForm = new ContactFormPageObject();
+describe('Demoblaze app', () => {
+  before(() => {
+    homeAndСatalogue.visit();
   });
 
-  it('', () => {
+  it('checking the ability to add and order goods', () => {
+    homeAndСatalogue.clickOnCategory('Laptop');
+    homeAndСatalogue.clickOnProduct('Sony vaio i7');
+    homeAndСatalogue.clickOnTheButton('Add to cart');
+    homeAndСatalogue.succsessfulAllert();
+    homeAndСatalogue.clickOnLink('Cart');
+
+    cartPage.assertAboutProduct('Sony vaio i7');
+    cartPage.clickOnTheButton('Place Order');
 
+    contactForm.typeName('Ivan');
```

Comment:
you can create testData obj in this file and generate random data what you will use below